src/server/models/users.model.js

```diff
@@ -66,9 +66,10 @@ let userSchema = new Schema({
   password: String,
   name: String,
   education: [educationSchema],
-  work: [workSchema],
-  job: [jobSchema],
-  help: [helpSchema],
+  experience: [experienceSchema],
+  work: [experienceSchema],
```

Comment:
Don't we get the facebook data and make another object named ```registrationData```  (user.controllers.js:245~246) ?

```
education: facebookResult.education,
experience: facebookResult.work,
```

Why the schema needs to use name ```work``` also?